Median_of_two_sorted_Array/median_ofTwoSortedArrays.cpp

```diff
@@ -0,0 +1,70 @@
+//median of two sorted arrays -
```

Comment:
Fix file and folder name.

Reply:
Okay , what should be a suitable name for this ?